mysql-test/suite/data_masking/t/gen_rnd_us_phone-master.opt

```diff
@@ -0,0 +1 @@
+$DATA_MASKING_PLUGIN_OPT $DATA_MASKING_PLUGIN_LOAD
```

Comment:
This could be `suite.opt` since all tests have the same master.opt in the suite.